client/js/components/user/CustomerSettings/CustomerSettingsMap.vue

```diff
@@ -104,30 +119,68 @@ export default {
 
   data () {
     return {
-      currentMapAttribution: this.mapAttribution,
-      layer: this.initLayer,
-      layerUrl: this.initLayerUrl,
+      mapAttribution: this.initMapAttribution,
+      baseLayerLayers: this.initLayer,
+      baseLayerUrl: this.initLayerUrl,
       mapKey: 0
     }
   },
 
   computed: {
+    ...mapState('customer', {
+      customerItems: 'items'
+    }),
+
     attributionControl () {
       return new Attribution({ collapsible: false })
     }
   },
 
   methods: {
+    ...mapActions('customer', {
+      saveCustomer: 'save'
+    }),
+
+    ...mapMutations('customer', {
+      updateCustomer: 'setItem'
+    }),
+
     debounceUpdate: debounce(({ id, value }) => {
       if (id === 'r_baseLayerUrl' || id === 'r_baseLayerLayers') {
         if (id === 'r_baseLayerUrl') {
-          this.layerUrl = value
+          this.baseLayerUrl = value
         } else if (id === 'r_baseLayerLayers') {
```

Comment:
this can be simplified since the `else-if` block is not needed.
You already check the ids here `id === 'r_baseLayerUrl' || id === 'r_baseLayerLayers'` therefore only one of both ids can be valid.

```suggestion
        } else {
```

Reply:
done https://github.com/demos-europe/demosplan-core/pull/2212/commits/384cd84b18fddb01a36f22d1dc04d580959cb013